.github/workflows/linting.yml

```diff
@@ -718,12 +718,16 @@ jobs:
           done
       - name: Run phan
         run: pnpm jetpack phan --all -v --update-baseline --format github
-      - name: Run phan for previous WP version too
+      - name: Run phan for previous WP version and old Woo
         env:
           # Don't bother complaining about unused suppressions that may be used with the newer stubs. See .phan/config.base.php for how this gets applied.
           NO_PHAN_UNUSED_SUPPRESSION: 1
+          # 2025-11-20: We have to draw the line somewhere. Let's go with Woo 7.0.
+          WOO_VERSION: 7
         run: |
-          composer update --prefer-lowest php-stubs/wordpress-stubs php-stubs/wordpress-tests-stubs
+          # We can't directly downgrade Woo stubs with `update` due to existing constraints
```

Comment:
Could we if we change the constraint in composer.json?

Reply:
Do you mean in `trunk` like we do with WP stubs? We could; I personally lean toward keeping constraints tighter in there, but right now it's `>=8.7`, so if that's your preference I can do that.